source/frontend/src/features/leases/detail/LeasePages/tenant/__snapshots__/AddLeaseTenantContainer.test.tsx.snap

```diff
@@ -171,7 +171,9 @@ exports[`AddLeaseTenantContainer component renders as expected 1`] = `
 }
 
 .c3 .table .tr .td:first-of-type,
-.c3 .table .tr .td:nth-of-type(2),.c3 .table .tr .th:first-of-type,.c3 .table .tr .th:nth-of-type(2) {
+.c3 .table .tr .td:nth-of-type(2),
+.c3 .table .tr .th:first-of-type,
+.c3 .table .tr .th:nth-of-type(2) {
```

Comment:
This seems unrelated to this pr but I guess we forgot to update them on a previous pr??

Reply:
apparently moving to this version of json5 changes the output of this snapshot.